main/guides/orchestration/txvsportfolio.md

```diff
@@ -0,0 +1,168 @@
+
+The `auto-stake-it` contract demonstrates how to continuously manage a user's positions. A user
+"onboards" into the contract by creating accounts on multiple chains (via orchestration) and then
+depositing tokens to be staked automatically.
+
+Key points in code:
+
+1. **Prepare a PortfolioHolder** that stores each user's accounts in a single structure.
+2. **Add accounts** to the portfolio. Each user might have multiple chain accounts.
+3. **Perform multi-step operations** (e.g. receive tokens over IBC, transfer to staking).
+
+Below is a simplified snippet (`auto-stake-it.flows.js`) showing how a portfolio is set up when a
+user calls the "makeAccounts" flow:
+
+```js
+export const makeAccounts = async (orch, { makeStakingTap, makePortfolioHolder, chainHub },
+    seat, { chainName, validator }) => {
+    seat.exit(); // no funds are exchanged at the moment
+
+    // 1. Get chain references (e.g., 'agoric' chain and the remote chain).
+    const [agoric, remoteChain] = await Promise.all([
+        orch.getChain('agoric'),
+        orch.getChain(chainName),
+    ]);
+
+    // 2. Create orchestration accounts for each chain.
+    const [localAccount, stakingAccount] = await Promise.all([
+        agoric.makeAccount(),
+        remoteChain.makeAccount(),
+    ]);
+
+    // 3. Combine them into a single PortfolioHolder for the user.
+    const accountEntries = harden([
+        ['agoric', localAccount],
+        [chainName, stakingAccount],
+    ]);
+    const publicTopicEntries = /* gather each account's public topics */;
+    const portfolioHolder = makePortfolioHolder(accountEntries, publicTopicEntries);
+
+    // 4. Return a continuing offer result with invitationMakers, etc.
+    return portfolioHolder.asContinuingOffer();
+};
+```
+
+In this snippet, we create multiple accounts (one for Agoric, one for the remote chain).
+Then, the contract uses `makePortfolioHolder` to maintain these in a persistent store.
+Users can perform multiple future actions without losing state. Learn more about the
+`makePortfolioHolder` function in the [`makePortfolioHolder` Orchestration API reference](https://agoric-sdk.pages.dev/funcs/_agoric_orchestration.preparePortfolioHolder).
+
+## Choosing Between Transactional and Portfolio
+
+Use **transactional contracts** (like "send-anywhere") when:
+
+- You want a **single-shot interaction** (e.g. a swap, a cross-chain send).
+- No ongoing user state is needed.
+- The user wants minimal overhead (no dedicated on-chain structure).
+
+Use **portfolio contracts** (like "auto-stake-it") when:
+
+- You need **long-lived, multi-step interactions** (e.g. staking, compounding).
+- Each user needs a separate sub-account or position.
+- User state must persist (e.g., "my tokens remain staked until I withdraw").
+
+Agoric Orchestration supports both **transactional contracts** for single interactions and
+**portfolio contracts** for rich, persistent user positions. Use the pattern that best fits
```

Comment:
The 1st sentence of this para does a good job of motivating the complexity. I wonder about moving it to the top of the page.